mobile/src/src/components/Community/Feed/EventView.jsx

```diff
@@ -67,56 +72,67 @@ const EventView = ({
             style={{ width: 0.88 * width, height: 300 }}
           />
         )}
-
         <View style={styles.buttonContainer}>
-          {is_attending === true ? (
+          {attending === true ? (
             <Button
               compact={true}
-              style={styles.buttonStyle}
+              style={styles.buttonStyleFade}
               mode="contained"
-              title="Attending"
-              color="green"
-              // todo handle press
-              // onPress={}
+              title="Unattend"
+              color="white"
+              dark={true}
+              onPress={() => {
+                attendEvent(id, !attending);
+                setAttending(!attending);
```

Comment:
As mentioned above, I don't think we need to manage state in this functional component.